File: PYTHON/Program/tempat_sampah.py
class TempatSampah:
    def __init__(self, id_tempat_sampah="", lokasi="", kapasitas=""):
        self.__id_tempat_sampah = id_tempat_sampah
        self.__lokasi = lokasi
        self.__kapasitas = kapasitas

    # getter setter id
    def get_id_tempat_sampah(self):
        return self.__id_tempat_sampah

    def set_id_tempat_sampah(self, id_tempat_sampah):
        self.__id_tempat_sampah = id_tempat_sampah

    # getter setter lokasi
    def get_lokasi(self):
        return self.__lokasi

    def set_lokasi(self, lokasi):
        self.__lokasi = lokasi

    # getter setter kapasitas
    def get_kapasitas(self):
        return self.__kapasitas

    def set_kapasitas(self, kapasitas):
        self.__kapasitas = kapasitas

File: CPP/Program/tempatSampah.cpp
#include <iostream>
#include <string>

using namespace std;

class TempatSampah {
    // Atribut private
    private:
        string idTempatSampah;
        string lokasi; 
        string kapasitas;

    public:
        // Constructor kosong
        TempatSampah() {
        }

        // Constructor
        TempatSampah(string idTempatSampah, string lokasi, string kapasitas) {
            this->idTempatSampah = idTempatSampah;
            this->lokasi = lokasi;
            this->kapasitas = kapasitas;
        }

        // getter setter id
        string getIdTempatSampah() {return idTempatSampah;}
        
        void setIdTempatSampah(string idTempatSampah) {this->idTempatSampah = idTempatSampah;}
        
        // getter setter lokasi
        string getLokasi() {return lokasi;}
        
        void setLokasi(string lokasi) {this->lokasi = lokasi;}
        
        // getter setter kapasitas
        string getKapasitas() {return kapasitas;}
        
        void setKapasitas(string kapasitas) {this->kapasitas = kapasitas;}
        
    // Destructor
    ~TempatSampah() {    
    }
};